src/NetscriptFunctions.ts

```diff
@@ -1740,7 +1740,7 @@ export const ns: InternalAPI<NSFull> = {
       (!hasTextExtension(sourcePath) && !hasScriptExtension(sourcePath)) ||
       (!hasTextExtension(destinationPath) && !hasScriptExtension(destinationPath))
     ) {
-      throw helpers.errorMessage(ctx, `'mv' can only be used on scripts and text files (.txt)`);
+      throw helpers.errorMessage(ctx, `'mv' can only be used on scripts and text files (.txt, .json)`);
```

Comment:
On the other hand, this is an error message so IMO it should also list the scripts.